libcontainer/specconv/spec_linux_test.go

```diff
@@ -546,6 +546,29 @@ func TestSpecconvExampleValidate(t *testing.T) {
 	}
 }
 
+func TestSpecconvNoLinuxSection(t *testing.T) {
+	spec := Example()
+	spec.Root.Path = "/"
+	spec.Linux = nil
+	spec.Hostname = ""
+
+	opts := &CreateOpts{
+		CgroupName:       "ContainerID",
+		UseSystemdCgroup: false,
+		Spec:             spec,
+	}
+
+	config, err := CreateLibcontainerConfig(opts)
+	if err != nil {
+		t.Errorf("Couldn't create libcontainer config: %v", err)
+	}
+
+	validator := validate.New()
+	if err := validator.Validate(config); err != nil {
+		t.Errorf("Expected specconv to produce valid container config: %v", err)
```

Comment:
```go
t.Errorf("Expected no error from validate, got %v", err)
```

Reply:
The existing code has consistent wording with the same checking in `TestSpecconvExampleValidate` and `TestNonZeroEUIDCompatibleSpecconvValidate`. You this this new case deserves different wording?

Reply:
Ah, OK then